View a list of comments associated with a review

PeterM24 opened the pull request:
Preview:
![image](https://user-images.githubusercontent.com/123666879/233052015-d39f2060-2769-4c20-873e-00becdd72ee4.png)

PeterM24 committed:
Create Comment interface:
- to pass to api Promise type
- to pass to comment.tsx files

PeterM24 committed:
Create get fetchComments api func

PeterM24 committed:
remove dummy logo  to stop errors

PeterM24 committed:
Create CommentsSingleReview component:
- returns a unordered list
- useEffect and useState handle api data fetching
- isLoading implemented for future loading comments feature

PeterM24 committed:
Render Comments on SingleReview page

PeterM24 committed:
Update CommentsSingleReview:
- better structure on page

PeterM24 committed:
format code

PeterM24 committed:
Update CSS for comments list

PeterM24 committed:
Add loading animation for comment fetching

popatre commented on src/components/CommentsSingleReview.tsx:
```diff
@@ -0,0 +1,50 @@
+          <div></div>
+          <div></div>
+          <div></div>
+          <div></div>
```
This div thing is a little messy - not sure if theres a cleaner way though

popatre commented on src/components/CommentsSingleReview.tsx:
```diff
@@ -0,0 +1,50 @@
+                <p className="comment-date">Posted: {date}</p>
+                <p className="comment-body">{comment.body}</p>
+                <p className="comment-votes">Votes: {comment.votes}</p>
+              </li>
```
Nice one - could you:

- [ ] Add a message to account for an article with no comments? Just so we get some feedback

popatre reviewed (approved):
Looks good, just needs:

- [ ] A message for when an article has no comments

PeterM24 committed:
Handle 0 comments:
- conditional statement loads user indication

Extract comment TSX
- into own IndividualComment tsx

Extract LoadingDots into its own file
- tidy up CommentsSingleReview component

PeterM24 committed:
Update Comments css for improved UX
